dev-qt/qtgui/qtgui-5.8.9999.ebuild

```diff
@@ -137,12 +138,14 @@ src_prepare() {
 	use dbus || sed -i -e 's/contains(QT_CONFIG, dbus)/false/' \
 		src/platformsupport/platformsupport.pro || die
 
+	qt_use_disable_config tuio udpsocket src/plugins/generic/generic.pro
```

Comment:
yeah it seems it's only used to determine the set of modules that need to be searched for the required feature, so we're good.